### DISCUSSION TITLE
Mouse support on soft keyboard with long press on space key?

### DISCUSSION THREAD

**Discussion by carstenf** [blinksh/blink#1881] [Ideas]
There is a mouse support using an external mouse.
If I use just the soft keyboard, a long press on the space key should give me as well mouse control.
It looks it’s not working on my iPad.
Is this an issue with my device or an issue with blink?

**Comment by carloscabanero** [comment]
This is not supported by Blink. The reason is because the soft keyboard and the mouse are different animals. The keyboard controls the caret, and we are not a regular TextView. We never really liked the implementations others made of this so instead we came up with better "arrow controls".

**Comment by carstenf** [comment]
The arrows are very useful, but the implementation in Pythonista for example feels more natural, for my liking.
Maybe could be an option for one of the next versions to have both?
